package.json

```diff
@@ -47,15 +55,16 @@
     "not Explorer <= 11"
   ],
   "devDependencies": {
-    "0x": "^5.7.0",
+    "@arethetypeswrong/cli": "^0.17.0",
     "@babel/core": "^7.26.0",
     "@babel/preset-env": "^7.26.0",
     "@babel/preset-typescript": "^7.26.0",
     "@rollup/plugin-babel": "^6.0.4",
     "@rollup/plugin-terser": "^0.4.4",
+    "0x": "^5.7.0",
     "jest": "^29.7.0",
     "rollup": "^4.26.0",
-    "typescript": "^5.6.3"
+    "typescript": "^5.7.1-rc"
```

Comment:
The 5.7 release candidate was necessary to gain support for `rewriteRelativeImportExtensions`, see https://devblogs.microsoft.com/typescript/announcing-typescript-5-7-rc/